.github/CODEOWNERS

```diff
@@ -1,5 +1,13 @@
-# API reports
-api-report/                                                               @microsoft/fluid-cr-api
+# Internal API reports
+api-report/                                                               @microsoft/fluid-cr-internal-api
```

Comment:
Would it be safer to reverse this?  I.e. suppose a new report directory is added, make it so by default it goes to fluid-cr-api and has to be explicitly changed to fluid-cr-internal-api if it's deemed internal.

Reply:
Hmmm, I think you're right. Given how CODEOWNERS works, we'll need to list each package explicitly OR create a separate path for the internal APIs.